.java-version

```diff
@@ -1 +1 @@
-11
+17
```

Comment:
See previous comment

Reply:
I upgrade the function runtime to java 17